Gruntfile.js

```diff
@@ -60,7 +60,7 @@ grunt.initConfig({
 		},
 		all: [
 			"<%= jshint.all %>",
-			"!test/deepEqual.js"
+			"!test/default/deepEqual.js"
```

Comment:
I feel like there's a better name for this, but can't come up with anything I like. `index` would match `index.html`, but doesn't make much sense otherwise. `main` or `core` isn't great either.